internal/services/automation/automation_runbook_resource.go

```diff
@@ -61,17 +63,10 @@ func resourceAutomationRunbook() *pluginsdk.Resource {
 			"resource_group_name": azure.SchemaResourceGroupName(),
 
 			"runbook_type": {
-				Type:     pluginsdk.TypeString,
-				Required: true,
-				ForceNew: true,
-				ValidateFunc: validation.StringInSlice([]string{
-					string(automation.RunbookTypeEnumGraph),
-					string(automation.RunbookTypeEnumGraphPowerShell),
-					string(automation.RunbookTypeEnumGraphPowerShellWorkflow),
-					string(automation.RunbookTypeEnumPowerShell),
-					string(automation.RunbookTypeEnumPowerShellWorkflow),
-					string(automation.RunbookTypeEnumScript),
-				}, false),
+				Type:         pluginsdk.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validation.StringInSlice(runbook.PossibleValuesForRunbookTypeEnum(), false),
```

Comment:
we should be explicit on the values here, that slice can change without warning due to bad swagger and then we would break users